### PR TITLE
Change version to 1.30.1-pre

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "__MSG_extensionName__",
   "description": "__MSG_extensionDescription__",
   "version": "1.31.0",
```
Comment (DNin01) [inline manifest.json:5]: Was this supposed to change to 1.30.1?

**Review by DNin01 — COMMENTED** [review]
I think you might have forgotten to change something?

File: manifest.json (hunk 1 above)
Comment (TheColaber) [reply manifest.json:5]: lol

Comment (WorldLanguages) [reply manifest.json:5]: Whoops

Comment (WorldLanguages) [reply manifest.json:5]: prereñease moment: #3834

**Pull request merged**